python/ray/experimental/sgd/pytorch/pytorch_trainer.py

```diff
@@ -11,12 +11,10 @@
 import ray
 
 from ray.tune import Trainable
-from ray.tune.resources import Resources
-from ray.experimental.sgd.pytorch.pytorch_runner import PyTorchRunner
-from ray.experimental.sgd.pytorch.distributed_pytorch_runner import (
-    DistributedPyTorchRunner)
-from ray.experimental.sgd.pytorch import pytorch_utils
-from ray.experimental.sgd import utils
+from ray.tune.trial import Resources
+from sgd.pytorch.pytorch_runner import PyTorchRunner
```

Comment:
everything here should be `ray.experimental.sgd...`

Reply:
should be fixed